prow/resultstore/payload.go

```diff
@@ -152,13 +152,28 @@ func (p *Payload) invocationProperties() []*resultstore.Property {
 			Value: p.Job.Status.URL,
```

Comment:
Adding this comment here because I don't think I can comment on a commit message directly. Anyway you have in your 2nd commit "Sort the values gotten from the yaml dict to
avoid testing flakes." but I think you meant to say that for the 1st commit.